project3/index.html

```diff
@@ -0,0 +1,129 @@
+            <fieldset>
+                <legend>Category</legend>
+                <span>
+                    <input name="category" id="alien" type="checkbox">
+                    <label for="alien">Alien</label>
+                </span>
+                <span>
+                    <input name="category" id="cyborg" type="checkbox">
+                    <label for="cyborg">Cyborg</label>
+                </span>
+                <span>
+                    <input name="category" id="experiment" type="checkbox">
+                    <label for="experiment">Experiment</label>
+                </span>
+                <span>
+                    <input name="category" id="human" type="checkbox">
+                    <label for="human">Human</label>
+                </span>
+                <span>
+                    <input name="category" id="inhuman" type="checkbox">
+                    <label for="inhuman">Inhuman</label>
+                </span>
+                <span>
+                    <input name="category" id="mutant" type="checkbox">
+                    <label for="mutant"> Mutant</label>
+                </span>
+
+            </fieldset>
+
+            <fieldset>
+                <legend>Gender</legend>
+                <span>
+                    <input name="gender" id="woman" type="radio">
+                    <label for="woman">Woman</label>
+                </span>
+                <span>
+
+                    <input name="gender" id="man" type="radio">
+                    <label for="man">Man</label>
+                </span>
+                <span>
+
+                    <input name="gender" id="nonbinary" type="radio">
+                    <label for="nonbinary">Nonbinary</label>
+                </span>
+                <span>
+
+                    <input name="gender" id="genderqueer" type="radio">
+                    <label for="genderqueer">Genderqueer/Genderfluid</label>
+                </span>
+                <span>
+
+                    <input name="gender" id="perfernottoanswer" type="radio">
+                    <label for="perfernottoanswer">Perfer not to answer</label>
+                </span>
+
+                <input name="gender" id="selfdescribe" type="radio">
+                <label for="selfdescribe">Prefer to self describe</label>
+
+                <label for="selfdescribe" id="selfdescribe">Describe Your Powers:</label>
+                <input name="gender" id="selfdescribe" type="text">
+                </span>
+
+            </fieldset>
+
+            <fieldset>
+                <legend>Powers and Abilities</legend>
+                <label for="powersandabliities" id="describepower">Describe Your Powers:</label>
+                <textarea name="describepower" id="message" required></textarea>
+
+                <legend>Rate Your Abilities:</legend>
+
+                <label for="strength">Strength:</label>
+                <select name="powersandabliities" id="Strength">
```

Comment:
Note that `id`s are case sensitive and the `name` and `id` of this element should match:
```suggestion
                <select name="strength" id="strength">
```